tests/tests_ntp_provider7.yml

```diff
@@ -0,0 +1,31 @@
+
+- name: Configure NTP as peer with chrony as specified provider
+  hosts: all
+  vars:
+    timesync_ntp_servers:
+      - hostname: 172.16.123.1
+    timesync_peer_server:
+      - hostname: 127.0.0.1
+    timesync_ntp_provider: chrony
+  roles:
+    - linux-system-roles.timesync
+
+  pre_tasks:
+    - name: Install chrony
+      package: name=chrony state=present
+      register: package_install
+      ignore_errors: yes
+
+    - meta: end_play
+      when: package_install.failed
+
+    - name: Remove chrony
+      package: name=chrony state=absent
+
+  post_tasks:
+    - name: Wait for services to start
+      wait_for:
+        timeout: 2
+
+    - name: Check chronyd service
+      shell: chronyc -n tracking
```

Comment:
please use the command module instead of shell if possible.

Reply:
TBH `tests_ntp_provider7.yml` is a simple copy of `tests_ntp_provider4.yml` with the additional param. Do you want me to change to command there as well? 

I'm not even sure where these tests get executed :/

Reply:
No, if it is a copy, we will change them all together.